File: rust/src/models/config_entitlement_value.rs
// this file is @generated
use serde::{Deserialize, Serialize};

use super::config_value::ConfigValue;

#[derive(Clone, Debug, Default, PartialEq, Deserialize, Serialize)]
pub struct ConfigEntitlementValue {
    pub value: ConfigValue,
}

impl ConfigEntitlementValue {
    pub fn new(value: ConfigValue) -> Self {
        Self { value }
    }
}

File: typescript/src/models/configEntitlementValue.ts
// this file is @generated
import { type ConfigValue, ConfigValueSerializer } from "./configValue";

export interface ConfigEntitlementValue {
  value: ConfigValue;
}

export const ConfigEntitlementValueSerializer = {
  _fromJsonObject(object: any): ConfigEntitlementValue {
    return {
      value: ConfigValueSerializer._fromJsonObject(object["value"]),
    };
  },

  _toJsonObject(self: ConfigEntitlementValue): any {
    return {
      value: ConfigValueSerializer._toJsonObject(self.value),
    };
  },
};
